### ISSUE TITLE
Termination criteria & Comparison measure

### ISSUE THREAD

**Issue by skjw1224** [skjw1224/epelRL#32]
- Performance measure
- Loss function
- Computational (cpu) time
- Amount of data
- Hyperparameter sensitivity
- Algorithmic complexity
- Exploitation & Exploration -- Regret
- ???
